P5/Exemplars/tei_jtei.odd

```diff
@@ -1993,9 +1993,9 @@
               <attDef ident="group" mode="delete"/>
             </attList>
           </classSpec>
+	  <!-- We should be removing @calendar here!! -->
```

Comment:
@sydb, this comment can go now, right?

Reply:
Uhhh … do we not need to do what it says, first? I mean these exemplars are not supposed to have a `@calendar` attribute.

Reply:
But it's done AFAIK!